tests/data/records/record-v1.0.0.json

```diff
@@ -1 +1,23 @@
-{"mappings": {}}
+{
```

Comment:
yes, because in the tests the OAISet was created before any record was inserted, elasticsearch was complaining that there was no field in the mapping corresponding to the fields involved in the percolator query.